cmake/FindROOT.cmake

```diff
@@ -198,7 +196,7 @@ function(REFLEX_GENERATE_DICTIONARY dictionary)
   #set(rootmapopts --rootmap=${rootmapname} --rootmap-lib=${libprefix}${dictionary}Dict)
   #---Actual command----------------------------------------
   add_custom_command(OUTPUT ${gensrcdict} ${rootmapname} ${dictionary}_rdict.pcm
-                     COMMAND ${LAUNCHER} ${GENREFLEX_EXECUTABLE} ${headerfiles} -o ${gensrcdict} ${rootmapopts} --select=${selectionfile} --noIncludePaths
+    COMMAND ${ROOTCLING_EXECUTABLE} -reflex -f ${gensrcdict} ${rootmapopts} ${headerfiles} ${selectionfile} -noIncludePaths -inlineInputHeader
```

Comment:
Here you removed the `--select` flag, however, in `bdm-code-generation` you kept it. Is this intentional?

Reply:
I forgot about it. Thank you for reminding me